app/src/main/graphql/project.graphql

```diff
@@ -1,28 +1,44 @@
 query FetchProjects(
-  $first: Int = 15,
-  $cursor: String,
-  $sort: ProjectSort,
-  $state: PublicProjectState,
-  $backed: Boolean,
-  $recommended: Boolean,
-  $categoryId: String,
-  $starred: Boolean,
-  $staffPicks: Boolean,
-  $searchTerm: String,
-  $raised: RaisedBuckets,
-  $location: ID) {
-  projects(first: $first, after:$cursor, sort:$sort, backed:$backed, recommended:$recommended, categoryId: $categoryId, starred:$starred,  state: $state, staffPicks: $staffPicks, term: $searchTerm, raised: $raised, locationId: $location) {
-    edges {
-      cursor
-      node{
-        ... projectCard
-      }
-    }
-    pageInfo {
-      ... pageInfo
+$first: Int = 15,
+$cursor: String,
+$sort: ProjectSort,
+$state: PublicProjectState,
+$backed: Boolean,
+$recommended: Boolean,
+$categoryId: String,
+$starred: Boolean,
+$staffPicks: Boolean,
+$searchTerm: String,
+$raised: RaisedBuckets,
+$location: ID,
+$amountRaisedBucket: PledgedBuckets
```

Comment:
Added `$amountRaisedBucket: PledgedBuckets` to the query, everything else is styling